src/pyscaffold/shell.py

```diff
@@ -132,10 +149,10 @@ def get_git_cmd(**args):
     Args:
         **args: additional keyword arguments to :obj:`~.ShellCommand`
     """
-    if sys.platform == "win32":  # pragma: no cover
+    if IS_WINDOWS:  # pragma: no cover
         # ^  CI setup does not aggregate Windows coverage
-        for cmd in ["git.cmd", "git.exe"]:
-            git = ShellCommand(cmd, **args)
+        for shell in (True, False):
+            git = ShellCommand("git.exe", shell=shell, **args)
```

Comment:
I am very curious to understand what is happening for the command with `shell=True` to be failing...

Would be a problem with file names?

Reply:
Yea, I am also very curious about this. We only now that one of @ksteptoe's two Windows machines seem to have a problem with the shell setup. Somehow an `&` is added but I have no good explanation for that. I think it will be impossible to figure this out without access to the machine. Also I am no Windows (Power)shell expert.